returns/functions.py

```diff
@@ -144,3 +148,12 @@ def raise_exception(exception: Exception) -> NoReturn:
 
     """
     raise exception
+
+
+def not_(
+    function: Callable[
+        [Result[_ValueType, _ErrorType]], Union[_Containers, bool],
+    ],
+) -> Callable[[Result[_ValueType, _ErrorType]], bool]:
+    """Denies the function return."""
+    return lambda container: not bool(function(container))
```

Comment:
`functools.wraps`?

Reply:
Maybe it is a great ideia, and to be more readable we have to re-write. It will look like this:
```python
def not_(function: Callable[..., bool]) -> Callable[..., bool]:
    """Denies the function return."""
    @wraps(function)
    def wrapped_function(*args, **kwargs):
        return not function(*args, **kwargs)
    return wrapped_function
```

Reply:
@sobolevn I think this is a great point!

Reply:
👍